pwiz_tools/Skyline/TestFunctional/PasteTransitionListTest.cs

```diff
@@ -315,5 +318,33 @@ private void TestPeptideOnlyDoc()
             RunUI(() => documentGrid.Close());
         }
 
+        private void TestMissingFragmentMz()
+        {
+            // Make sure we properly handle missing fragment mz info
+            var saveColumnOrder = Settings.Default.CustomMoleculeTransitionInsertColumnsList;
+            var text =  "Peptide Modified Sequence\tPrecursor m/z\tFragment Name\tProduct Charge\nPEPTIDER\t478.738\ty1\t1\nPEPTIDER\t478.738\ty2\t1";

```

Comment:
I think these numbers are going to need to be formatted with the localized decimal separator.

Reply:
FWIW that happens on line 328.